TestFramework/Tests.Unit/Tests/UserUnitTests.cs

```diff
@@ -140,7 +140,8 @@ public async Task TestMICValidateAuthServiceID()
 		}
 
 		[Test]
-		public async Task TestMICRenderURLScopeID()
+        [Obsolete("This test has been deprecated.")]
+        public async Task TestMICRenderURLScopeID()
```

Comment:
We are not deprecating the normal MIC flow (Authorization Grant Flow), why is this test method being deprecated?